change/@fluentui-react-jsx-runtime-d2a505fc-6ffc-4679-aa63-5b29dcc83f7b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1767 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28373&commitId=1aecbda274a1f0f03aa4e1f876b883ece6ca9492&env=prod)

✅ There was 0 screenshots added, 844 screenshots removed, 187 screenshots unchanged, 0 screenshots with different dimensions and 923 screenshots with visible difference.


  <details><summary><b>unknown</b> 1767 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|234828| Changed|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|237185| Changed|
|Accordion Converged.disabled.normal.chromium.png|262101| Changed|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|235492| Changed|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|237435| Changed|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|261468| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|261474| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|256077| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|256073| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|261823| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|261813| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|287459| Changed|
|Accordion Converged.size.normal.chromium.png|287369| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|234370| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|237039| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|256073| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|256040| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|275954| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|275841| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|231698| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|237508| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|257813| Changed|
|Avatar Converged.basic.normal.chromium.png|257802| Changed|
|Avatar Converged.color - Dark Mode.normal.chromium.png|244974| Changed|
|Avatar Converged.color - High Contrast.normal.chromium.png|243850| Changed|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|250110| Changed|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|250824| Changed|
|Avatar Converged.color+active.normal.chromium.png|354195| Changed|
|Avatar Converged.color.normal.chromium.png|362495| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|317843| Changed|
|Avatar Converged.customSize+image.normal.chromium.png|314101| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|319962| Changed|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|236825| Changed|
|Avatar Converged.image-bad-url.normal.chromium.png|236814| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|381955| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|371304| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|378292| Changed|
|Avatar Converged.size+name.normal.chromium.png|383473| Changed|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|224857| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|268254| Changed|
|AvatarGroup Converged.basic.default.chromium.png|715548| Changed|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|243461| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|256915| Changed|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|338011| Changed|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|237906| Changed|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|238291| Changed|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|239596| Changed|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|239596| Changed|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|237948| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|238356| Changed|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|239593| Changed|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|239601| Changed|
|AvatarGroup Converged.layoutPie.default.chromium.png|337931| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|232380| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|266482| Changed|
|AvatarGroup Converged.layoutStack.default.chromium.png|618219| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|1596541| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|1598289| Changed|
|AvatarGroup Converged.overflowContent.chromium.png|1605603| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|223764| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|267335| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|716657| Changed|
|Badge Converged.filled - Dark Mode.chromium.png|1267174| Changed|
|Badge Converged.filled - High Contrast.chromium.png|1257895| Changed|
|Badge Converged.filled.chromium.png|1590909| Changed|
|Badge Converged.ghost - Dark Mode.chromium.png|1261411| Changed|
|Badge Converged.ghost - High Contrast.chromium.png|1241290| Changed|
|Badge Converged.ghost.chromium.png|1598750| Changed|
|Badge Converged.outline - Dark Mode.chromium.png|1263965| Changed|
|Badge Converged.outline - High Contrast.chromium.png|1245744| Changed|
|Badge Converged.outline.chromium.png|1595993| Changed|
|Badge Converged.size- extra-large - RTL.chromium.png|1581606| Changed|
|Badge Converged.size- extra-large.chromium.png|1584040| Changed|
|Badge Converged.size- extra-small - RTL.chromium.png|1604605| Changed|
|Badge Converged.size- extra-small.chromium.png|1604712| Changed|
|Badge Converged.size- large - RTL.chromium.png|1588673| Changed|
|Badge Converged.size- large.chromium.png|1590029| Changed|
|Badge Converged.size- medium - RTL.chromium.png|1591642| Changed|
|Badge Converged.size- medium.chromium.png|1592531| Changed|
|Badge Converged.size- small - RTL.chromium.png|1595700| Changed|
|Badge Converged.size- small.chromium.png|1596205| Changed|
|Badge Converged.tint - Dark Mode.chromium.png|1260889| Changed|
|Badge Converged.tint - High Contrast.chromium.png|1245138| Changed|
|Badge Converged.tint.chromium.png|1601347| Changed|
|Badge Converged.width constrained.chromium.png|1604068| Changed|
|Button Converged.As an anchor.default.chromium.png|1606227| Changed|
|Button Converged.Circular and icon only - RTL.default.chromium.png|1606925| Changed|
|Button Converged.Circular and icon only.default.chromium.png|1606925| Changed|
|Button Converged.Circular.default.chromium.png|1606232| Changed|
|Button Converged.Default - Dark Mode.default.chromium.png|1563843| Changed|
|Button Converged.Default - High Contrast.default.chromium.png|1572599| Changed|
|Button Converged.Default - RTL.default.chromium.png|1606226| Changed|
|Button Converged.Default.default.chromium.png|1606227| Changed|
|Button Converged.Disabled - Dark Mode.default.chromium.png|1563798| Changed|
|Button Converged.Disabled - High Contrast.default.chromium.png|1572405| Changed|
|Button Converged.Disabled.default.chromium.png|1606271| Changed|
|Button Converged.Icon only.default.chromium.png|1606921| Changed|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|1563630| Changed|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|1572405| Changed|
|Button Converged.Outline Disabled.default.chromium.png|1606432| Changed|
|Button Converged.Outline.default.chromium.png|1606227| Changed|
|Button Converged.Primary - Dark Mode.default.chromium.png|1563889| Changed|
|Button Converged.Primary - High Contrast.default.chromium.png|1573809| Changed|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|1563818| Changed|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|1572377| Changed|
|Button Converged.Primary Disabled.default.chromium.png|1606287| Changed|
|Button Converged.Primary.default.chromium.png|1604602| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|1605098| Changed|
|Button Converged.Size large.default.chromium.png|1606476| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|1605501| Changed|
|Button Converged.Size small - with long text wrapping.chromium.png|1606214| Changed|
|Button Converged.Size small.default.chromium.png|1606769| Changed|
|Button Converged.Subtle - Dark Mode.default.chromium.png|1563785| Changed|
|Button Converged.Subtle - High Contrast.default.chromium.png|1572483| Changed|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|1563631| Changed|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|1572377| Changed|
|Button Converged.Subtle Disabled.default.chromium.png|1606464| Changed|
|Button Converged.Subtle.default.chromium.png|1606310| Changed|
|Button Converged.Transparent - Dark Mode.default.chromium.png|1563785| Changed|
|Button Converged.Transparent - High Contrast.default.chromium.png|1572483| Changed|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|1563631| Changed|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|1572377| Changed|
|Button Converged.Transparent Disabled.default.chromium.png|1606464| Changed|
|Button Converged.Transparent.default.chromium.png|1606310| Changed|
|Button Converged.With icon after content - RTL.default.chromium.png|1606605| Changed|
|Button Converged.With icon after content.default.chromium.png|1606606| Changed|
|Button Converged.With icon before content - RTL.default.chromium.png|1606605| Changed|
|Button Converged.With icon before content.default.chromium.png|1606606| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|238489| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|241706| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|238261| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|241706| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|268544| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|269414| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|238485| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|241706| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|269247| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|238488| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|241706| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|269415| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|270027| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|239115| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|242316| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|270220| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|238890| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|242316| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|269351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|269158| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|270027| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|239101| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|242316| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|270054| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|269862| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|239114| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|242316| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|270221| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|270029| Changed|
|Card Converged.CardHeader.normal.chromium.png|377056| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|210061| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|259737| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|612584| Changed|
|Card Converged.appearance.normal.chromium.png|612409| Changed|
|Card Converged.card templates.normal.chromium.png|345157| Changed|
|Card Converged.orientation.chromium.png|1598000| Changed|
|Card Converged.size.normal.chromium.png|447934| Changed|
|Checkbox Converged.checked.rest.chromium.png|245385| Changed|
|Checkbox Converged.circular+checked.default.chromium.png|245306| Changed|
|Checkbox Converged.circular+mixed.default.chromium.png|245391| Changed|
|Checkbox Converged.circular.default.chromium.png|245574| Changed|
|Checkbox Converged.disabled+checked.default.chromium.png|245586| Changed|
|Checkbox Converged.disabled+mixed.default.chromium.png|245587| Changed|
|Checkbox Converged.disabled.rest.chromium.png|245633| Changed|
|Checkbox Converged.label-before - RTL.default.chromium.png|245504| Changed|
|Checkbox Converged.label-before.default.chromium.png|245504| Changed|
|Checkbox Converged.label-wrapping - RTL.default.chromium.png|257584| Changed|
|Checkbox Converged.label-wrapping.default.chromium.png|257575| Changed|
|Checkbox Converged.large - RTL.default.chromium.png|246155| Changed|
|Checkbox Converged.large+checked.default.chromium.png|245788| Changed|
|Checkbox Converged.large+circular+checked.default.chromium.png|245730| Changed|
|Checkbox Converged.large+circular+mixed.default.chromium.png|245838| Changed|
|Checkbox Converged.large+circular.default.chromium.png|246084| Changed|
|Checkbox Converged.large+label-wrapping - RTL.default.chromium.png|258773| Changed|
|Checkbox Converged.large+label-wrapping.default.chromium.png|258773| Changed|
|Checkbox Converged.large+mixed.default.chromium.png|245933| Changed|
|Checkbox Converged.large.default.chromium.png|246155| Changed|
|Checkbox Converged.mixed.rest.chromium.png|245504| Changed|
|Checkbox Converged.no-label.default.chromium.png|245655| Changed|
|Checkbox Converged.required+label-before.default.chromium.png|245331| Changed|
|Checkbox Converged.required.default.chromium.png|245531| Changed|
|Checkbox Converged.unchecked - RTL.rest.chromium.png|245520| Changed|
|Checkbox Converged.unchecked.rest.chromium.png|245520| Changed|
|Combobox Converged.Appearance- filled-darker.default.chromium.png|244507| Changed|
|Combobox Converged.Appearance- filled-lighter.default.chromium.png|244818| Changed|
|Combobox Converged.Appearance- outline (default).default.chromium.png|245482| Changed|
|Combobox Converged.Appearance- underline.default.chromium.png|245316| Changed|
|Combobox Converged.Disabled option.default.chromium.png|273479| Changed|
|Combobox Converged.Disabled with value.default.chromium.png|245610| Changed|
|Combobox Converged.Disabled.default.chromium.png|245629| Changed|
|Combobox Converged.Invalid- filled-darker.default.chromium.png|244191| Changed|
|Combobox Converged.Invalid- filled-lighter.default.chromium.png|244480| Changed|
|Combobox Converged.Invalid- outline.default.chromium.png|245355| Changed|
|Combobox Converged.Invalid- underline.default.chromium.png|245319| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|317751| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|273447| Changed|
|Combobox Converged.Open.default.chromium.png|273447| Changed|
|Combobox Converged.Option with long content.default.chromium.png|301506| Changed|
|Combobox Converged.Size- large.default.chromium.png|246295| Changed|
|Combobox Converged.Size- small.default.chromium.png|243693| Changed|
|Combobox Converged.With multiselect selection.default.chromium.png|273107| Changed|
|Combobox Converged.With placeholder.default.chromium.png|245426| Changed|
|Combobox Converged.With selection.default.chromium.png|273429| Changed|
|Combobox Converged.With value.default.chromium.png|245351| Changed|
|CompoundButton Converged.Circular and icon only - RTL.default.chromium.png|1606273| Changed|
|CompoundButton Converged.Circular and icon only.default.chromium.png|1606274| Changed|
|CompoundButton Converged.Circular.default.chromium.png|1605473| Changed|
|CompoundButton Converged.Default - RTL.default.chromium.png|1605454| Changed|
|CompoundButton Converged.Default.default.chromium.png|1605465| Changed|
|CompoundButton Converged.Disabled.default.chromium.png|1605297| Changed|
|CompoundButton Converged.Icon only.default.chromium.png|1606266| Changed|
|CompoundButton Converged.Outline Disabled.default.chromium.png|1605858| Changed|
|CompoundButton Converged.Outline.default.chromium.png|1605465| Changed|
|CompoundButton Converged.Primary Disabled.default.chromium.png|1605324| Changed|
|CompoundButton Converged.Primary.default.chromium.png|1599864| Changed|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|1604542| Changed|
|CompoundButton Converged.Size large.default.chromium.png|1605672| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|1604687| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|1605080| Changed|
|CompoundButton Converged.Size small.default.chromium.png|1605741| Changed|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|1605912| Changed|
|CompoundButton Converged.Subtle.default.chromium.png|1605585| Changed|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|1605912| Changed|
|CompoundButton Converged.Transparent.default.chromium.png|1605585| Changed|
|CompoundButton Converged.With icon after content - RTL.default.chromium.png|1605644| Changed|
|CompoundButton Converged.With icon after content.default.chromium.png|1605728| Changed|
|CompoundButton Converged.With icon before content - RTL.default.chromium.png|1605644| Changed|
|CompoundButton Converged.With icon before content.default.chromium.png|1605726| Changed|
|DatePicker Compat.DateRange- day.default.chromium.png|379957| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|264165| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|261293| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|377338| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|379452| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|379629| Changed|
|DatePicker Compat.Required.default.chromium.png|392950| Changed|
|DatePicker Compat.Underlined and required.default.chromium.png|383054| Changed|
|DatePicker Compat.Underlined.default.chromium.png|383054| Changed|
|DatePicker Compat.With label.default.chromium.png|392957| Changed|
|DatePicker Compat.allowTextInput.default.chromium.png|383004| Changed|
|DatePicker Compat.default - Dark Mode.rest.chromium.png|249308| Changed|
|DatePicker Compat.default - High Contrast.rest.chromium.png|246192| Changed|
|DatePicker Compat.default.rest.chromium.png|383004| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|251117| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|249897| Changed|
|DatePicker Compat.marked dates.default.chromium.png|379958| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|251135| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|250178| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|379814| Changed|
|Dialog.Fluid Actions End.chromium.png|912604| Changed|
|Dialog.Fluid Actions Start.chromium.png|912603| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|810366| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|833615| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|913161| Changed|
|Dialog.actions position start & position end.chromium.png|913162| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|810007| Changed|
|Dialog.actions position start - High Contrast.chromium.png|833229| Changed|
|Dialog.actions position start - RTL.chromium.png|913493| Changed|
|Dialog.actions position start.chromium.png|913494| Changed|
|Dialog.alert - Dark Mode.chromium.png|810007| Changed|
|Dialog.alert - High Contrast.chromium.png|833227| Changed|
|Dialog.alert - RTL.chromium.png|913492| Changed|
|Dialog.alert.chromium.png|913493| Changed|
|Dialog.default - Dark Mode.chromium.png|810007| Changed|
|Dialog.default - High Contrast.chromium.png|833227| Changed|
|Dialog.default - RTL.chromium.png|913493| Changed|
|Dialog.default.chromium.png|913493| Changed|
|Dialog.nested - Dark Mode.chromium.png|814183| Changed|
|Dialog.nested - High Contrast.chromium.png|835507| Changed|
|Dialog.nested - RTL.chromium.png|907397| Changed|
|Dialog.nested.chromium.png|907397| Changed|
|Dialog.no actions - Dark Mode.chromium.png|820172| Changed|
|Dialog.no actions - High Contrast.chromium.png|838569| Changed|
|Dialog.no actions - RTL.chromium.png|906420| Changed|
|Dialog.no actions.chromium.png|906421| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|827543| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|843817| Changed|
|Dialog.no title & no actions - RTL.chromium.png|900328| Changed|
|Dialog.no title & no actions.chromium.png|900328| Changed|
|Dialog.no title - Dark Mode.chromium.png|817378| Changed|
|Dialog.no title - High Contrast.chromium.png|838476| Changed|
|Dialog.no title - RTL.chromium.png|907400| Changed|
|Dialog.no title.chromium.png|907401| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|1465595| Changed|
|Dialog.non-modal - High Contrast.chromium.png|1490332| Changed|
|Dialog.non-modal - RTL.chromium.png|1595993| Changed|
|Dialog.non-modal.chromium.png|1595994| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|665846| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|704642| Changed|
|Dialog.scroll long content - RTL.chromium.png|1052344| Changed|
|Dialog.scroll long content.chromium.png|1052384| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|819396| Changed|
|Dialog.title custom action - High Contrast.chromium.png|837976| Changed|
|Dialog.title custom action - RTL.chromium.png|907020| Changed|
|Dialog.title custom action.chromium.png|907020| Changed|
|Divider Converged - Horizontal.Appearance brand - Dark Mode.default.chromium.png|239157| Changed|
|Divider Converged - Horizontal.Appearance brand - High Contrast.default.chromium.png|239955| Changed|
|Divider Converged - Horizontal.Appearance brand.default.chromium.png|241819| Changed|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|239141| Changed|
|Divider Converged - Horizontal.Appearance strong - High Contrast.default.chromium.png|239955| Changed|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|241926| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|239158| Changed|
|Divider Converged - Horizontal.Appearance subtle - High Contrast.default.chromium.png|239955| Changed|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|241955| Changed|
|Divider Converged - Horizontal.End Aligned - RTL.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.End Aligned.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.Inset.default.chromium.png|241943| Changed|
|Divider Converged - Horizontal.Start Aligned - RTL.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.Start Aligned.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.with content - Dark Mode.default.chromium.png|239138| Changed|
|Divider Converged - Horizontal.with content - High Contrast.default.chromium.png|239955| Changed|
|Divider Converged - Horizontal.with content - RTL.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.with content.default.chromium.png|241940| Changed|
|Divider Converged - Horizontal.without content - RTL.default.chromium.png|238624| Changed|
|Divider Converged - Horizontal.without content.default.chromium.png|238624| Changed|
|Divider Converged - Vertical.Center Aligned.default.chromium.png|239309| Changed|
|Divider Converged - Vertical.End Aligned.default.chromium.png|239309| Changed|
|Divider Converged - Vertical.Start Aligned.default.chromium.png|239309| Changed|
|Divider Converged - Vertical.inset.default.chromium.png|240102| Changed|
|Dropdown Converged.Appearance- filled-darker.default.chromium.png|244502| Changed|
|Dropdown Converged.Appearance- filled-lighter.default.chromium.png|244813| Changed|
|Dropdown Converged.Appearance- outline (default).default.chromium.png|245477| Changed|
|Dropdown Converged.Appearance- underline.default.chromium.png|245311| Changed|
|Dropdown Converged.Disabled option.default.chromium.png|245453| Changed|
|Dropdown Converged.Disabled with value.default.chromium.png|245604| Changed|
|Dropdown Converged.Disabled.default.chromium.png|245623| Changed|
|Dropdown Converged.Invalid- filled-darker.default.chromium.png|244185| Changed|
|Dropdown Converged.Invalid- filled-lighter.default.chromium.png|244475| Changed|
|Dropdown Converged.Invalid- outline.default.chromium.png|245349| Changed|
|Dropdown Converged.Invalid- underline.default.chromium.png|245313| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|245453| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|245453| Changed|
|Dropdown Converged.Open.default.chromium.png|245453| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|245453| Changed|
|Dropdown Converged.Size- large.default.chromium.png|246304| Changed|
|Dropdown Converged.Size- small.default.chromium.png|243912| Changed|
|Dropdown Converged.With multiselect selection.default.chromium.png|245453| Changed|
|Dropdown Converged.With multiselect value.default.chromium.png|245295| Changed|
|Dropdown Converged.With placeholder.default.chromium.png|245336| Changed|
|Dropdown Converged.With selection.default.chromium.png|245453| Changed|
|Dropdown Converged.With value.default.chromium.png|245345| Changed|
|Field.Checkbox.default.chromium.png|273918| Changed|
|Field.Combobox.default.chromium.png|324683| Changed|
|Field.Dropdown.default.chromium.png|324687| Changed|
|Field.ProgressBar.default.chromium.png|302990| Changed|
|Field.RadioGroup.default.chromium.png|324381| Changed|
|Field.Select.default.chromium.png|324995| Changed|
|Field.Slider.default.chromium.png|280472| Changed|
|Field.SpinButton.default.chromium.png|324284| Changed|
|Field.Switch.default.chromium.png|285029| Changed|
|Field.Textarea.default.chromium.png|354440| Changed|
|Field.base.default.chromium.png|253550| Changed|
|Field.hint.default.chromium.png|264566| Changed|
|Field.horizontal+longLabel.default.chromium.png|274722| Changed|
|Field.horizontal+noLabel.default.chromium.png|261467| Changed|
|Field.horizontal.default.chromium.png|248074| Changed|
|Field.required.default.chromium.png|253534| Changed|
|Field.size-large.default.chromium.png|257058| Changed|
|Field.size-small.default.chromium.png|250242| Changed|
|Field.validation-error.default.chromium.png|259947| Changed|
|Field.validation-none.default.chromium.png|260067| Changed|
|Field.validation-success.default.chromium.png|273858| Changed|
|Field.validation-warning.default.chromium.png|260052| Changed|
|Field.validationMessageIcon.default.chromium.png|259725| Changed|
|FluentProvider CustomStyleHooks.Button.chromium.png|1604004| Changed|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|1601419| Changed|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|1603219| Changed|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|1603751| Changed|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|1603571| Changed|
|InfoButton.default - Dark Mode.focus.chromium.png|239344| Changed|
|InfoButton.default - Dark Mode.rest.chromium.png|239194| Changed|
|InfoButton.default - High Contrast.focus.chromium.png|241412| Changed|
|InfoButton.default - High Contrast.rest.chromium.png|241342| Changed|
|InfoButton.default.focus.chromium.png|260986| Changed|
|InfoButton.default.rest.chromium.png|261126| Changed|
|InfoButton.size.rest.chromium.png|307957| Changed|
|InfoLabel.default - Dark Mode.rest.chromium.png|239230| Changed|
|InfoLabel.default - High Contrast.rest.chromium.png|240002| Changed|
|InfoLabel.default - RTL.rest.chromium.png|242629| Changed|
|InfoLabel.default.rest.chromium.png|242628| Changed|
|InfoLabel.required - RTL.rest.chromium.png|242513| Changed|
|InfoLabel.required.rest.chromium.png|242508| Changed|
|InfoLabel.size-large.rest.chromium.png|242738| Changed|
|InfoLabel.size-small.rest.chromium.png|242668| Changed|
|InfoLabel.wrap.rest.chromium.png|248915| Changed|
|Input Converged.Appearance- filled-darker.default.chromium.png|245252| Changed|
|Input Converged.Appearance- filled-lighter.default.chromium.png|248110| Changed|
|Input Converged.Appearance- outline (default).default.chromium.png|245324| Changed|
|Input Converged.Appearance- underline.default.chromium.png|245370| Changed|
|Input Converged.Disabled, appearance- filled-darker.default.chromium.png|245524| Changed|
|Input Converged.Disabled, appearance- filled-lighter.default.chromium.png|247915| Changed|
|Input Converged.Disabled, appearance- outline.default.chromium.png|245524| Changed|
|Input Converged.Disabled, appearance- underline.default.chromium.png|245555| Changed|
|Input Converged.Inline.default.chromium.png|249900| Changed|
|Input Converged.Invalid, appearance- filled-darker.default.chromium.png|244994| Changed|
|Input Converged.Invalid, appearance- filled-lighter.default.chromium.png|247834| Changed|
|Input Converged.Invalid, appearance- outline.default.chromium.png|245217| Changed|
|Input Converged.Invalid, appearance- underline.default.chromium.png|245372| Changed|
|Input Converged.Size- large.default.chromium.png|246088| Changed|
|Input Converged.Size- small.default.chromium.png|243594| Changed|
|Input Converged.With appearance override.default.chromium.png|253305| Changed|
|Input Converged.With value.default.chromium.png|245354| Changed|
|Input Converged.contentAfter - RTL.default.chromium.png|245289| Changed|
|Input Converged.contentAfter.default.chromium.png|245289| Changed|
|Input Converged.contentBefore - RTL.default.chromium.png|245284| Changed|
|Input Converged.contentBefore.default.chromium.png|245284| Changed|
|Label Converged.CustomRequired - RTL.default.chromium.png|1606522| Changed|
|Label Converged.CustomRequired.default.chromium.png|1606524| Changed|
|Label Converged.Multiline - RTL.default.chromium.png|1606254| Changed|
|Label Converged.Multiline.default.chromium.png|1606250| Changed|
|Label Converged.Required - Dark Mode.default.chromium.png|1583786| Changed|
|Label Converged.Required - High Contrast.default.chromium.png|1589376| Changed|
|Label Converged.Required.default.chromium.png|1606752| Changed|
|Menu Converged - basic.default - RTL.chromium.png|1605984| Changed|
|Menu Converged - basic.default.chromium.png|1605992| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|1554003| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|1559378| Changed|
|Menu Converged - groups.default - RTL.chromium.png|1605130| Changed|
|Menu Converged - groups.default.chromium.png|1605144| Changed|
|Menu Converged - icon slotted content.default - RTL.chromium.png|1605777| Changed|
|Menu Converged - icon slotted content.default.chromium.png|1605794| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|1605666| Changed|
|Menu Converged - nested submenus.default.chromium.png|1605685| Changed|
|Menu Converged - secondary content.default - RTL.chromium.png|1605845| Changed|
|Menu Converged - secondary content.default.chromium.png|1605861| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|1605058| Changed|
|Menu Converged - selection groups.default.chromium.png|1605109| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|1571054| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|1577496| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|1605921| Changed|
|Menu Converged - selection.checkbox.chromium.png|1605942| Changed|
|Menu Converged - split item.default - RTL.chromium.png|1605750| Changed|
|Menu Converged - split item.default.chromium.png|1605769| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|1606243| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|1606257| Changed|
|Menu nested within a MenuTrigger.default.chromium.png|1605184| Changed|
|Menu.Nested Submenus Small Viewport Flipped.default.chromium.png|1604517| Changed|
|Menu.Nested Submenus Small Viewport Stacked.default.chromium.png|1605007| Changed|
|MenuButton Converged.Circular and icon only.default.chromium.png|1606531| Changed|
|MenuButton Converged.Circular.default.chromium.png|1605833| Changed|
|MenuButton Converged.Default - RTL.default.chromium.png|1605827| Changed|
|MenuButton Converged.Default.default.chromium.png|1605828| Changed|
|MenuButton Converged.Disabled.default.chromium.png|1605851| Changed|
|MenuButton Converged.Icon only.default.chromium.png|1606526| Changed|
|MenuButton Converged.Outline Disabled.default.chromium.png|1606040| Changed|
|MenuButton Converged.Outline.default.chromium.png|1605828| Changed|
|MenuButton Converged.Primary Disabled.default.chromium.png|1605869| Changed|
|MenuButton Converged.Primary.default.chromium.png|1603890| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|1604701| Changed|
|MenuButton Converged.Size large.default.chromium.png|1606065| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|1605101| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|1605788| Changed|
|MenuButton Converged.Size small.default.chromium.png|1606363| Changed|
|MenuButton Converged.Subtle Disabled.default.chromium.png|1606076| Changed|
|MenuButton Converged.Subtle.default.chromium.png|1605918| Changed|
|MenuButton Converged.Transparent Disabled.default.chromium.png|1606076| Changed|
|MenuButton Converged.Transparent.default.chromium.png|1605918| Changed|
|MenuButton Converged.With icon.default.chromium.png|1606200| Changed|
|Persona Converged.basic - Dark Mode.normal.chromium.png|233393| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|237384| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|306319| Changed|
|Persona Converged.basic.normal.chromium.png|306292| Changed|
|Persona Converged.size-avatar.normal.chromium.png|495966| Changed|
|Persona Converged.size-presence.normal.chromium.png|489912| Changed|
|Persona Converged.textAlignment.normal.chromium.png|316094| Changed|
|Persona Converged.textPosition.normal.chromium.png|283994| Changed|
|Persona Converged.textWrap.normal.chromium.png|361570| Changed|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|1606451| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|1606452| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.default.chromium.png|304470| Changed|
|PresenceBadge Converged - inverted background.default - High Contrast.default.chromium.png|254150| Changed|
|PresenceBadge Converged - inverted background.default.default.chromium.png|253789| Changed|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|1606109| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|1606108| Changed|
|PresenceBadge Converged - status.default - RTL.chromium.png|1606264| Changed|
|PresenceBadge Converged - status.default.chromium.png|1606264| Changed|
|ProgressBar converged.Determinate - Dark Mode.chromium.png|1605122| Changed|
|ProgressBar converged.Determinate - High Contrast.chromium.png|1605746| Changed|
|ProgressBar converged.Determinate - RTL.chromium.png|1605492| Changed|
|ProgressBar converged.Determinate with thickness large.chromium.png|1604089| Changed|
|ProgressBar converged.Determinate.chromium.png|1605492| Changed|
|ProgressBar converged.Error.chromium.png|1605554| Changed|
|ProgressBar converged.Indeterminate - Dark Mode.chromium.png|1604950| Changed|
|ProgressBar converged.Indeterminate - High Contrast.chromium.png|1605560| Changed|
|ProgressBar converged.Indeterminate - RTL.chromium.png|1606179| Changed|
|ProgressBar converged.Indeterminate with thickness large.chromium.png|1605461| Changed|
|ProgressBar converged.Indeterminate.chromium.png|1606179| Changed|
|ProgressBar converged.Success.chromium.png|1605567| Changed|
|ProgressBar converged.Warning.chromium.png|1605558| Changed|
|Radio Converged.checked - Dark Mode.rest.chromium.png|238867| Changed|
|Radio Converged.checked - High Contrast.rest.chromium.png|240159| Changed|
|Radio Converged.checked.rest.chromium.png|245410| Changed|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|238720| Changed|
|Radio Converged.disabled - High Contrast.rest.chromium.png|240080| Changed|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|238776| Changed|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|240143| Changed|
|Radio Converged.disabled+checked.default.chromium.png|245518| Changed|
|Radio Converged.disabled.rest.chromium.png|245580| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|259956| Changed|
|Radio Converged.group-disabled.default.chromium.png|259956| Changed|
|Radio Converged.group-horizontal - RTL.default.chromium.png|245161| Changed|
|Radio Converged.group-horizontal-stacked - RTL.default.chromium.png|251107| Changed|
|Radio Converged.group-horizontal-stacked.default.chromium.png|251055| Changed|
|Radio Converged.group-horizontal.default.chromium.png|245161| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|259685| Changed|
|Radio Converged.group-vertical.default.chromium.png|259677| Changed|
|Radio Converged.label-after - RTL.default.chromium.png|245481| Changed|
|Radio Converged.label-after.default.chromium.png|245481| Changed|
|Radio Converged.label-below - RTL.default.chromium.png|251325| Changed|
|Radio Converged.label-below.default.chromium.png|251318| Changed|
|Radio Converged.label-wrapping - RTL.default.chromium.png|257538| Changed|
|Radio Converged.label-wrapping.default.chromium.png|257529| Changed|
|Radio Converged.no-label - RTL.default.chromium.png|245609| Changed|
|Radio Converged.no-label.default.chromium.png|245609| Changed|
|Radio Converged.required - RTL.default.chromium.png|245491| Changed|
|Radio Converged.required.default.chromium.png|245491| Changed|
|Radio Converged.unchecked - Dark Mode.rest.chromium.png|238808| Changed|
|Radio Converged.unchecked - High Contrast.rest.chromium.png|240164| Changed|
|Radio Converged.unchecked.rest.chromium.png|245474| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|245141| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|245487| Changed|
|Select Converged.Appearance- outline (default).default.chromium.png|245307| Changed|
|Select Converged.Appearance- underline.default.chromium.png|245371| Changed|
|Select Converged.Custom Icon.default.chromium.png|245300| Changed|
|Select Converged.Disabled.default.chromium.png|245554| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|244790| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|245112| Changed|
|Select Converged.Invalid- outline.default.chromium.png|245159| Changed|
|Select Converged.Invalid- underline.default.chromium.png|245373| Changed|
|Select Converged.Size- large.default.chromium.png|246115| Changed|
|Select Converged.Size- small.default.chromium.png|243535| Changed|
|Select Converged.With appearance override.default.chromium.png|253001| Changed|
|Select Converged.With value.default.chromium.png|245269| Changed|
|Slider Converged.Disabled - Dark Mode.default.chromium.png|237751| Changed|
|Slider Converged.Disabled - High Contrast.default.chromium.png|238479| Changed|
|Slider Converged.Disabled Vertical.default.chromium.png|239447| Changed|
|Slider Converged.Disabled.default.chromium.png|240543| Changed|
|Slider Converged.Horizontal - 0% - RTL.default.chromium.png|240297| Changed|
|Slider Converged.Horizontal - 0%.default.chromium.png|240297| Changed|
|Slider Converged.Horizontal - 100% - RTL.default.chromium.png|240286| Changed|
|Slider Converged.Horizontal - 100%.default.chromium.png|240285| Changed|
|Slider Converged.Root - Dark Mode.default.chromium.png|237920| Changed|
|Slider Converged.Root - High Contrast.default.chromium.png|238692| Changed|
|Slider Converged.Root - RTL.default.chromium.png|240320| Changed|
|Slider Converged.Root.default.chromium.png|240320| Changed|
|Slider Converged.Vertical - 0% - RTL.default.chromium.png|239201| Changed|
|Slider Converged.Vertical - 0%.default.chromium.png|239201| Changed|
|Slider Converged.Vertical - 100% - RTL.default.chromium.png|239191| Changed|
|Slider Converged.Vertical - 100%.default.chromium.png|239191| Changed|
|Slider Converged.Vertical - RTL.default.chromium.png|239225| Changed|
|Slider Converged.Vertical.default.chromium.png|239225| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|239216| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|240364| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|245385| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|245385| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|242518| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|243742| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|248280| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|248280| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|238870| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|240364| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|245470| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|245470| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|238904| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|240287| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|245517| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|245517| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|238868| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|240362| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|245473| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|245474| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|238867| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|240360| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|245473| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|245474| Changed|
|SpinButton Converged.Custom Width - Dark Mode.chromium.png|1598995| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|1600323| Changed|
|SpinButton Converged.Custom Width - RTL.chromium.png|1606820| Changed|
|SpinButton Converged.Custom Width.chromium.png|1606821| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|238759| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|240183| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|245627| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|245627| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|238952| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|240411| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|245390| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|245390| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|245113| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|247990| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|245358| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|245519| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|245470| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|245470| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|239081| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|240234| Changed|
|SpinButton Converged.Size- small - RTL.default.chromium.png|243711| Changed|
|SpinButton Converged.Size- small.default.chromium.png|243712| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|253786| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|1605525| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|1605444| Changed|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|1604824| Changed|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|1604685| Changed|
|Spinner converged.Inverted with Huge.chromium.png|1604824| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|1603633| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|1603435| Changed|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|1603213| Changed|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|1602630| Changed|
|Spinner converged.Inverted with Label Above.chromium.png|1603213| Changed|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|1603633| Changed|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|1603435| Changed|
|Spinner converged.Inverted with Label After.chromium.png|1603633| Changed|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|1603623| Changed|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|1603420| Changed|
|Spinner converged.Inverted with Label Before.chromium.png|1603623| Changed|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|1603206| Changed|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|1602618| Changed|
|Spinner converged.Inverted with Label Below.chromium.png|1603206| Changed|
|Spinner converged.Inverted with Label.chromium.png|1603633| Changed|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|1605031| Changed|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|1604909| Changed|
|Spinner converged.Inverted with Size Extra Large.chromium.png|1605031| Changed|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|1605908| Changed|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|1605852| Changed|
|Spinner converged.Inverted with Size Extra Small.chromium.png|1605908| Changed|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|1605288| Changed|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|1605189| Changed|
|Spinner converged.Inverted with Size Large.chromium.png|1605288| Changed|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|1605525| Changed|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|1605444| Changed|
|Spinner converged.Inverted with Size Medium.chromium.png|1605525| Changed|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|1605706| Changed|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|1605642| Changed|
|Spinner converged.Inverted with Size Small.chromium.png|1605706| Changed|
|Spinner converged.Inverted.chromium.png|1605525| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|1606260| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|1606482| Changed|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|1605670| Changed|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|1606076| Changed|
|Spinner converged.Primary with Huge.chromium.png|1606683| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|1604648| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|1605092| Changed|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|1604377| Changed|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|1604892| Changed|
|Spinner converged.Primary with Label Above.chromium.png|1606566| Changed|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|1604648| Changed|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|1605092| Changed|
|Spinner converged.Primary with Label After - RTL.chromium.png|1606547| Changed|
|Spinner converged.Primary with Label After.chromium.png|1606544| Changed|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|1604653| Changed|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|1605097| Changed|
|Spinner converged.Primary with Label Before - RTL.chromium.png|1606544| Changed|
|Spinner converged.Primary with Label Before.chromium.png|1606547| Changed|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|1604384| Changed|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|1604911| Changed|
|Spinner converged.Primary with Label Below.chromium.png|1606547| Changed|
|Spinner converged.Primary with Label.chromium.png|1606544| Changed|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|1605843| Changed|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|1606164| Changed|
|Spinner converged.Primary with Size Extra Large.chromium.png|1606769| Changed|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|1606581| Changed|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|1606704| Changed|
|Spinner converged.Primary with Size Extra Small.chromium.png|1606898| Changed|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|1606057| Changed|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|1606328| Changed|
|Spinner converged.Primary with Size Large.chromium.png|1606789| Changed|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|1606260| Changed|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|1606482| Changed|
|Spinner converged.Primary with Size Medium.chromium.png|1606813| Changed|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|1606416| Changed|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|1606574| Changed|
|Spinner converged.Primary with Size Small.chromium.png|1606884| Changed|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|1606064| Changed|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|1606010| Changed|
|Spinner converged.Primary with Size Tiny.chromium.png|1606064| Changed|
|Spinner converged.Primary.chromium.png|1606813| Changed|
|Switch Converged.Disabled and checked - Dark Mode.default.chromium.png|237773| Changed|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|238541| Changed|
|Switch Converged.Disabled and checked.default.chromium.png|241489| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.default.chromium.png|237748| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|238541| Changed|
|Switch Converged.Disabled and unchecked.default.chromium.png|241510| Changed|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|238083| Changed|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|238780| Changed|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|241044| Changed|
|Switch Converged.Enabled and checked.default.chromium.png|241044| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.default.chromium.png|237999| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|238710| Changed|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|241244| Changed|
|Switch Converged.Enabled and unchecked.default.chromium.png|241243| Changed|
|Switch Converged.Required.default.chromium.png|241516| Changed|
|Switch Converged.With label above - RTL.default.chromium.png|240908| Changed|
|Switch Converged.With label above.default.chromium.png|240905| Changed|
|Switch Converged.With label after - RTL.default.chromium.png|241244| Changed|
|Switch Converged.With label after.default.chromium.png|241243| Changed|
|Switch Converged.With label before - RTL.default.chromium.png|241243| Changed|
|Switch Converged.With label before.default.chromium.png|241244| Changed|
|Switch Converged.With label wrapping.default.chromium.png|260576| Changed|
|TabList and Tab Converged.Default - Dark Mode.default.chromium.png|1544739| Changed|
|TabList and Tab Converged.Default - High Contrast.default.chromium.png|1556451| Changed|
|TabList and Tab Converged.Default - RTL.default.chromium.png|1606833| Changed|
|TabList and Tab Converged.Default.default.chromium.png|1606833| Changed|
|TabList and Tab Converged.Large size - Dark Mode.default.chromium.png|1531915| Changed|
|TabList and Tab Converged.Large size - High Contrast.default.chromium.png|1546245| Changed|
|TabList and Tab Converged.Large size.default.chromium.png|1606759| Changed|
|TabList and Tab Converged.Small size - Dark Mode.default.chromium.png|1564310| Changed|
|TabList and Tab Converged.Small size - High Contrast.default.chromium.png|1572979| Changed|
|TabList and Tab Converged.Small size.default.chromium.png|1606838| Changed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.default.chromium.png|1544739| Changed|
|TabList and Tab Converged.Subtle appearance - High Contrast.default.chromium.png|1556451| Changed|
|TabList and Tab Converged.Subtle appearance.default.chromium.png|1606833| Changed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.default.chromium.png|1544880| Changed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.default.chromium.png|1556561| Changed|
|TabList and Tab Converged.Tab Selected (default).default.chromium.png|1606663| Changed|
|TabList and Tab Converged.Tab Selected - Dark Mode.default.chromium.png|1544880| Changed|
|TabList and Tab Converged.Tab Selected - High Contrast.default.chromium.png|1556561| Changed|
|TabList and Tab Converged.Tab Selected.default.chromium.png|1606663| Changed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.default.chromium.png|1439579| Changed|
|TabList and Tab Converged.Vertical and large size - High Contrast.default.chromium.png|1448824| Changed|
|TabList and Tab Converged.Vertical and large size.default.chromium.png|1606788| Changed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.default.chromium.png|1502979| Changed|
|TabList and Tab Converged.Vertical and small size - High Contrast.default.chromium.png|1521891| Changed|
|TabList and Tab Converged.Vertical and small size.default.chromium.png|1606839| Changed|
|TabList and Tab Converged.Vertical.default.chromium.png|1606870| Changed|
|TabList and Tab Converged.With icon - RTL.default.chromium.png|1606692| Changed|
|TabList and Tab Converged.With icon and vertical - RTL.default.chromium.png|1606694| Changed|
|TabList and Tab Converged.With icon and vertical.default.chromium.png|1606742| Changed|
|TabList and Tab Converged.With icon only and vertical.default.chromium.png|1606939| Changed|
|TabList and Tab Converged.With icon only.default.chromium.png|1606938| Changed|
|TabList and Tab Converged.With icon.default.chromium.png|1606692| Changed|
|Table flex - subtle selection.rest.chromium.png|1599960| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|1303828| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|1297768| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|1599810| Changed|
|Table layout flex - cell actions.always visible.chromium.png|1599912| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|1303703| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|1297611| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|1599987| Changed|
|Table layout flex - cell actions.default.chromium.png|1600066| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|1600066| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|1303744| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|1297621| Changed|
|Table layout flex - headers.sortable.chromium.png|1600027| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|1601009| Changed|
|Table layout flex - truncate.false.chromium.png|1601009| Changed|
|Table layout flex - truncate.true.chromium.png|1603756| Changed|
|Table layout flex.keyboard column resizing style.chromium.png|1596158| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|1295367| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|1295853| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|1582908| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|1582907| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|1315954| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|1460867| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|1574958| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|1574949| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|1297621| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|1296891| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|1591355| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|1591361| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|1308895| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|1386846| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|1587358| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|1587361| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|1303884| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|1297888| Changed|
|Table layout flex.multiselect - RTL.chromium.png|1599850| Changed|
|Table layout flex.multiselect.chromium.png|1599859| Changed|
|Table layout flex.primary cell.chromium.png|1598951| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|1303858| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|1297799| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|1599874| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|1300432| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|1297203| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|1595766| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|1595776| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|1306323| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|1343243| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|1593765| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|1593774| Changed|
|Table layout flex.single select (checked).chromium.png|1599883| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|1303832| Changed|
|Table layout flex.single select - High Contrast.chromium.png|1297785| Changed|
|Table layout flex.single select - RTL.chromium.png|1599922| Changed|
|Table layout flex.single select.chromium.png|1599930| Changed|
|Table layout flex.size - extra small.chromium.png|1602787| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|1303703| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|1297611| Changed|
|Table layout flex.size - medium - RTL.chromium.png|1599987| Changed|
|Table layout flex.size - medium.chromium.png|1600066| Changed|
|Table layout flex.size - small.chromium.png|1600444| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|1303828| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|1297768| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|1599810| Changed|
|Table layout table - cell actions.always visible.chromium.png|1599912| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|1303703| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|1297611| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|1599987| Changed|
|Table layout table - cell actions.default.chromium.png|1600066| Changed|
|Table layout table - cell actions.in header cell.chromium.png|1600066| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|1303744| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|1297621| Changed|
|Table layout table - headers.sortable.chromium.png|1600027| Changed|
|Table layout table - truncate.default (disabled).chromium.png|1601009| Changed|
|Table layout table - truncate.false.chromium.png|1601009| Changed|
|Table layout table - truncate.true.chromium.png|1603756| Changed|
|Table layout table.keyboard column resizing style.chromium.png|1595920| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|1295643| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|1295853| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|1583298| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|1583296| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|1315954| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|1460867| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|1574958| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|1574949| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|1297743| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|1296891| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|1591550| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|1591556| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|1308895| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|1386846| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|1587358| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|1587361| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|1303884| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|1297888| Changed|
|Table layout table.multiselect - RTL.chromium.png|1599850| Changed|
|Table layout table.multiselect.chromium.png|1599859| Changed|
|Table layout table.primary cell.chromium.png|1599039| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|1303858| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|1297799| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|1599874| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|1300515| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|1297203| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|1595863| Changed|
|Table layout table.single select (checked) - brand.chromium.png|1595873| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|1306323| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|1343243| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|1593765| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|1593774| Changed|
|Table layout table.single select (checked).chromium.png|1599883| Changed|
|Table layout table.single select - Dark Mode.chromium.png|1303832| Changed|
|Table layout table.single select - High Contrast.chromium.png|1297785| Changed|
|Table layout table.single select - RTL.chromium.png|1599922| Changed|
|Table layout table.single select.chromium.png|1599930| Changed|
|Table layout table.size - extra small.chromium.png|1602787| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|1303703| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|1297611| Changed|
|Table layout table.size - medium - RTL.chromium.png|1599987| Changed|
|Table layout table.size - medium.chromium.png|1600066| Changed|
|Table layout table.size - small.chromium.png|1600444| Changed|
|Table table - subtle selection.rest.chromium.png|1599960| Changed|
|Textarea Converged.Appearance- filled-darker.default.chromium.png|248977| Changed|
|Textarea Converged.Appearance- filled-lighter.default.chromium.png|254517| Changed|
|Textarea Converged.Appearance- outline (default).default.chromium.png|249231| Changed|
|Textarea Converged.Disabled + filled.default.chromium.png|249368| Changed|
|Textarea Converged.Disabled + placeholder.default.chromium.png|249333| Changed|
|Textarea Converged.Disabled.default.chromium.png|249368| Changed|
|Textarea Converged.Invalid- filled-darker.default.chromium.png|248697| Changed|
|Textarea Converged.Invalid- filled-lighter.default.chromium.png|254209| Changed|
|Textarea Converged.Invalid- outline.default.chromium.png|249110| Changed|
|Textarea Converged.Size- large.default.chromium.png|252308| Changed|
|Textarea Converged.Size- medium.default.chromium.png|249231| Changed|
|Textarea Converged.Size- small.default.chromium.png|246622| Changed|
|Textarea Converged.With appearance override.default.chromium.png|265147| Changed|
|Textarea Converged.With value.default.chromium.png|249270| Changed|
|Toast.Full Toast.chromium.png|1606423| Changed|
|Toast.Title Only.chromium.png|1606423| Changed|
|Toast.Without Subtitle.chromium.png|1606423| Changed|
|ToggleButton Converged.Checked.default.chromium.png|1605578| Changed|
|ToggleButton Converged.Circular and icon only.default.chromium.png|1606485| Changed|
|ToggleButton Converged.Circular.default.chromium.png|1605798| Changed|
|ToggleButton Converged.Default - Dark Mode.default.chromium.png|1563409| Changed|
|ToggleButton Converged.Default - High Contrast.default.chromium.png|1572165| Changed|
|ToggleButton Converged.Default - RTL.default.chromium.png|1605793| Changed|
|ToggleButton Converged.Default.default.chromium.png|1605794| Changed|
|ToggleButton Converged.Disabled.default.chromium.png|1605838| Changed|
|ToggleButton Converged.Icon only.default.chromium.png|1606480| Changed|
|ToggleButton Converged.Outline.default.chromium.png|1605794| Changed|
|ToggleButton Converged.Primary - Dark Mode.default.chromium.png|1563456| Changed|
|ToggleButton Converged.Primary - High Contrast.default.chromium.png|1573376| Changed|
|ToggleButton Converged.Primary Checked.default.chromium.png|1604160| Changed|
|ToggleButton Converged.Primary Disabled.default.chromium.png|1605854| Changed|
|ToggleButton Converged.Primary.default.chromium.png|1604168| Changed|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|1604657| Changed|
|ToggleButton Converged.Size large.default.chromium.png|1606035| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|1605060| Changed|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|1605773| Changed|
|ToggleButton Converged.Size small.default.chromium.png|1606328| Changed|
|ToggleButton Converged.Subtle Checked.default.chromium.png|1605605| Changed|
|ToggleButton Converged.Subtle Disabled.default.chromium.png|1606031| Changed|
|ToggleButton Converged.Subtle.default.chromium.png|1605877| Changed|
|ToggleButton Converged.Transparent Checked.default.chromium.png|1605904| Changed|
|ToggleButton Converged.Transparent Disabled.default.chromium.png|1606031| Changed|
|ToggleButton Converged.Transparent.default.chromium.png|1605877| Changed|
|ToggleButton Converged.With icon after content.default.chromium.png|1606165| Changed|
|ToggleButton Converged.With icon before content.default.chromium.png|1606165| Changed|
|Toolbar Converged.Default.default.chromium.png|248899| Changed|
|Toolbar Converged.Far Group.chromium.png|1603446| Changed|
|Toolbar Converged.Large.default.chromium.png|248898| Changed|
|Toolbar Converged.Small.default.chromium.png|245467| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|251714| Changed|
|Toolbar Converged.Vertical.default.chromium.png|291479| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|1580663| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|1581604| Changed|
|Tooltip Converged.basic.chromium.png|1606401| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|1580870| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|1581267| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|1603239| Changed|
|Tooltip Converged.inverted.chromium.png|1604925| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|1605770| Changed|
|Tooltip Converged.overflow-wrap.chromium.png|1605733| Changed|
|Tooltip Converged.positioning - High Contrast.chromium.png|1440701| Changed|
|Tooltip Converged.positioning - RTL.chromium.png|1606385| Changed|
|Tooltip Converged.positioning.chromium.png|1606385| Changed|
|Tooltip Converged.text-wrapping.chromium.png|1605797| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|1580755| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|1581655| Changed|
|Tooltip Converged.withArrow.chromium.png|1606311| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|0| Removed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|0| Removed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|0| Removed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|0| Removed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|0| Removed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|0| Removed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|0| Removed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|0| Removed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|0| Removed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|0| Removed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|0| Removed|
|Button Converged.As an anchor.hover.chromium.png|0| Removed|
|Button Converged.As an anchor.pressed.chromium.png|0| Removed|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|0| Removed|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|0| Removed|
|Button Converged.Circular and icon only.hover.chromium.png|0| Removed|
|Button Converged.Circular and icon only.pressed.chromium.png|0| Removed|
|Button Converged.Circular.hover.chromium.png|0| Removed|
|Button Converged.Circular.pressed.chromium.png|0| Removed|
|Button Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Default - RTL.hover.chromium.png|0| Removed|
|Button Converged.Default - RTL.pressed.chromium.png|0| Removed|
|Button Converged.Default.hover.chromium.png|0| Removed|
|Button Converged.Default.pressed.chromium.png|0| Removed|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Disabled.hover.chromium.png|0| Removed|
|Button Converged.Disabled.pressed.chromium.png|0| Removed|
|Button Converged.Icon only.hover.chromium.png|0| Removed|
|Button Converged.Icon only.pressed.chromium.png|0| Removed|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Outline Disabled.hover.chromium.png|0| Removed|
|Button Converged.Outline Disabled.pressed.chromium.png|0| Removed|
|Button Converged.Outline.hover.chromium.png|0| Removed|
|Button Converged.Outline.pressed.chromium.png|0| Removed|
|Button Converged.Primary - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Primary - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Primary - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Primary Disabled.hover.chromium.png|0| Removed|
|Button Converged.Primary Disabled.pressed.chromium.png|0| Removed|
|Button Converged.Primary.hover.chromium.png|0| Removed|
|Button Converged.Primary.pressed.chromium.png|0| Removed|
|Button Converged.Size large.hover.chromium.png|0| Removed|
|Button Converged.Size large.pressed.chromium.png|0| Removed|
|Button Converged.Size small.hover.chromium.png|0| Removed|
|Button Converged.Size small.pressed.chromium.png|0| Removed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Subtle Disabled.hover.chromium.png|0| Removed|
|Button Converged.Subtle Disabled.pressed.chromium.png|0| Removed|
|Button Converged.Subtle.hover.chromium.png|0| Removed|
|Button Converged.Subtle.pressed.chromium.png|0| Removed|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Transparent - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Transparent Disabled.hover.chromium.png|0| Removed|
|Button Converged.Transparent Disabled.pressed.chromium.png|0| Removed|
|Button Converged.Transparent.hover.chromium.png|0| Removed|
|Button Converged.Transparent.pressed.chromium.png|0| Removed|
|Button Converged.With icon after content - RTL.hover.chromium.png|0| Removed|
|Button Converged.With icon after content - RTL.pressed.chromium.png|0| Removed|
|Button Converged.With icon after content.hover.chromium.png|0| Removed|
|Button Converged.With icon after content.pressed.chromium.png|0| Removed|
|Button Converged.With icon before content - RTL.hover.chromium.png|0| Removed|
|Button Converged.With icon before content - RTL.pressed.chromium.png|0| Removed|
|Button Converged.With icon before content.hover.chromium.png|0| Removed|
|Button Converged.With icon before content.pressed.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|0| Removed|
|Checkbox Converged.checked.active.chromium.png|0| Removed|
|Checkbox Converged.checked.hover.chromium.png|0| Removed|
|Checkbox Converged.disabled.active.chromium.png|0| Removed|
|Checkbox Converged.disabled.hover.chromium.png|0| Removed|
|Checkbox Converged.mixed.active.chromium.png|0| Removed|
|Checkbox Converged.mixed.hover.chromium.png|0| Removed|
|Checkbox Converged.unchecked - RTL.active.chromium.png|0| Removed|
|Checkbox Converged.unchecked - RTL.hover.chromium.png|0| Removed|
|Checkbox Converged.unchecked.active.chromium.png|0| Removed|
|Checkbox Converged.unchecked.hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Combobox Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Combobox Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled option.active option.chromium.png|0| Removed|
|Combobox Converged.Disabled option.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled with value.focused.chromium.png|0| Removed|
|Combobox Converged.Disabled with value.hover.chromium.png|0| Removed|
|Combobox Converged.Disabled.focused.chromium.png|0| Removed|
|Combobox Converged.Disabled.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Combobox Converged.Invalid- underline.focused.chromium.png|0| Removed|
|Combobox Converged.Invalid- underline.hover.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Combobox Converged.Open.active option.chromium.png|0| Removed|
|Combobox Converged.Open.hover.chromium.png|0| Removed|
|Combobox Converged.Option with long content.active option.chromium.png|0| Removed|
|Combobox Converged.Option with long content.hover.chromium.png|0| Removed|
|Combobox Converged.Size- large.focused.chromium.png|0| Removed|
|Combobox Converged.Size- large.hover.chromium.png|0| Removed|
|Combobox Converged.Size- small.focused.chromium.png|0| Removed|
|Combobox Converged.Size- small.hover.chromium.png|0| Removed|
|Combobox Converged.With placeholder.focused.chromium.png|0| Removed|
|Combobox Converged.With placeholder.hover.chromium.png|0| Removed|
|Combobox Converged.With selection.active option.chromium.png|0| Removed|
|Combobox Converged.With selection.hover.chromium.png|0| Removed|
|Combobox Converged.With value.focused.chromium.png|0| Removed|
|Combobox Converged.With value.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Circular.hover.chromium.png|0| Removed|
|CompoundButton Converged.Circular.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Default.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Icon only.hover.chromium.png|0| Removed|
|CompoundButton Converged.Icon only.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Outline.hover.chromium.png|0| Removed|
|CompoundButton Converged.Outline.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Primary.hover.chromium.png|0| Removed|
|CompoundButton Converged.Primary.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Size large.hover.chromium.png|0| Removed|
|CompoundButton Converged.Size large.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Size small.hover.chromium.png|0| Removed|
|CompoundButton Converged.Size small.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Subtle.hover.chromium.png|0| Removed|
|CompoundButton Converged.Subtle.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|0| Removed|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Transparent.hover.chromium.png|0| Removed|
|CompoundButton Converged.Transparent.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|0| Removed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|0| Removed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|0| Removed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|0| Removed|
|DatePicker Compat.default.opened.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.active option.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Dropdown Converged.Open.active option.chromium.png|0| Removed|
|Dropdown Converged.Open.hover.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.active option.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.hover.chromium.png|0| Removed|
|Dropdown Converged.With selection.active option.chromium.png|0| Removed|
|Dropdown Converged.With selection.hover.chromium.png|0| Removed|
|InfoButton.default - Dark Mode.active.chromium.png|0| Removed|
|InfoButton.default - Dark Mode.hover.chromium.png|0| Removed|
|InfoButton.default - High Contrast.active.chromium.png|0| Removed|
|InfoButton.default - High Contrast.hover.chromium.png|0| Removed|
|InfoButton.default.active.chromium.png|0| Removed|
|InfoButton.default.hover.chromium.png|0| Removed|
|Input Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Input Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Input Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- outline.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- outline.hover.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Disabled, appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-darker.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-darker.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-lighter.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- filled-lighter.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- outline.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- outline.hover.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- underline.focused.chromium.png|0| Removed|
|Input Converged.Invalid, appearance- underline.hover.chromium.png|0| Removed|
|Input Converged.With value.focused.chromium.png|0| Removed|
|Input Converged.With value.hover.chromium.png|0| Removed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - basic.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|0| Removed|
|Menu Converged - nested submenus.default.all open.chromium.png|0| Removed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection groups.default.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox.selected.chromium.png|0| Removed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|0| Removed|
|Menu Converged - split item.default.submenu open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|0| Removed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|0| Removed|
|MenuButton Converged.Circular and icon only.hover.chromium.png|0| Removed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|0| Removed|
|MenuButton Converged.Circular.hover.chromium.png|0| Removed|
|MenuButton Converged.Circular.pressed.chromium.png|0| Removed|
|MenuButton Converged.Default - RTL.hover.chromium.png|0| Removed|
|MenuButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|MenuButton Converged.Default.hover.chromium.png|0| Removed|
|MenuButton Converged.Default.pressed.chromium.png|0| Removed|
|MenuButton Converged.Disabled.hover.chromium.png|0| Removed|
|MenuButton Converged.Disabled.pressed.chromium.png|0| Removed|
|MenuButton Converged.Icon only.hover.chromium.png|0| Removed|
|MenuButton Converged.Icon only.pressed.chromium.png|0| Removed|
|MenuButton Converged.Outline Disabled.hover.chromium.png|0| Removed|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|0| Removed|
|MenuButton Converged.Outline.hover.chromium.png|0| Removed|
|MenuButton Converged.Outline.pressed.chromium.png|0| Removed|
|MenuButton Converged.Primary Disabled.hover.chromium.png|0| Removed|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|0| Removed|
|MenuButton Converged.Primary.hover.chromium.png|0| Removed|
|MenuButton Converged.Primary.pressed.chromium.png|0| Removed|
|MenuButton Converged.Size large.hover.chromium.png|0| Removed|
|MenuButton Converged.Size large.pressed.chromium.png|0| Removed|
|MenuButton Converged.Size small.hover.chromium.png|0| Removed|
|MenuButton Converged.Size small.pressed.chromium.png|0| Removed|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|0| Removed|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|0| Removed|
|MenuButton Converged.Subtle.hover.chromium.png|0| Removed|
|MenuButton Converged.Subtle.pressed.chromium.png|0| Removed|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|0| Removed|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|0| Removed|
|MenuButton Converged.Transparent.hover.chromium.png|0| Removed|
|MenuButton Converged.Transparent.pressed.chromium.png|0| Removed|
|MenuButton Converged.With icon.hover.chromium.png|0| Removed|
|MenuButton Converged.With icon.pressed.chromium.png|0| Removed|
|Radio Converged.checked - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.checked - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.checked - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.checked - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.checked.active.chromium.png|0| Removed|
|Radio Converged.checked.hover.chromium.png|0| Removed|
|Radio Converged.disabled - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.disabled - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.disabled - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.disabled.active.chromium.png|0| Removed|
|Radio Converged.disabled.hover.chromium.png|0| Removed|
|Radio Converged.unchecked - Dark Mode.active.chromium.png|0| Removed|
|Radio Converged.unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Radio Converged.unchecked - High Contrast.active.chromium.png|0| Removed|
|Radio Converged.unchecked - High Contrast.hover.chromium.png|0| Removed|
|Radio Converged.unchecked.active.chromium.png|0| Removed|
|Radio Converged.unchecked.hover.chromium.png|0| Removed|
|Select Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Select Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Select Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Select Converged.Appearance- underline.focused.chromium.png|0| Removed|
|Select Converged.Appearance- underline.hover.chromium.png|0| Removed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Select Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Select Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Select Converged.Invalid- underline.focused.chromium.png|0| Removed|
|Select Converged.Invalid- underline.hover.chromium.png|0| Removed|
|Slider Converged.Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Slider Converged.Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled - High Contrast.hover.chromium.png|0| Removed|
|Slider Converged.Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled Vertical.hover.chromium.png|0| Removed|
|Slider Converged.Disabled Vertical.pressed.chromium.png|0| Removed|
|Slider Converged.Disabled.hover.chromium.png|0| Removed|
|Slider Converged.Disabled.pressed.chromium.png|0| Removed|
|Slider Converged.Root - Dark Mode.hover.chromium.png|0| Removed|
|Slider Converged.Root - Dark Mode.pressed.chromium.png|0| Removed|
|Slider Converged.Root - High Contrast.hover.chromium.png|0| Removed|
|Slider Converged.Root - High Contrast.pressed.chromium.png|0| Removed|
|Slider Converged.Root - RTL.hover.chromium.png|0| Removed|
|Slider Converged.Root - RTL.pressed.chromium.png|0| Removed|
|Slider Converged.Root.hover.chromium.png|0| Removed|
|Slider Converged.Root.pressed.chromium.png|0| Removed|
|Slider Converged.Vertical - RTL.hover.chromium.png|0| Removed|
|Slider Converged.Vertical - RTL.pressed.chromium.png|0| Removed|
|Slider Converged.Vertical.hover.chromium.png|0| Removed|
|Slider Converged.Vertical.pressed.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.focused.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and checked.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and checked.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked.hover.chromium.png|0| Removed|
|Switch Converged.Disabled and unchecked.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and checked.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and checked.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - Dark Mode.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - Dark Mode.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked.hover.chromium.png|0| Removed|
|Switch Converged.Enabled and unchecked.pressed.chromium.png|0| Removed|
|Switch Converged.Required.hover.chromium.png|0| Removed|
|Switch Converged.Required.pressed.chromium.png|0| Removed|
|Switch Converged.With label above - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label above - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label above.hover.chromium.png|0| Removed|
|Switch Converged.With label above.pressed.chromium.png|0| Removed|
|Switch Converged.With label after - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label after - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label after.hover.chromium.png|0| Removed|
|Switch Converged.With label after.pressed.chromium.png|0| Removed|
|Switch Converged.With label before - RTL.hover.chromium.png|0| Removed|
|Switch Converged.With label before - RTL.pressed.chromium.png|0| Removed|
|Switch Converged.With label before.hover.chromium.png|0| Removed|
|Switch Converged.With label before.pressed.chromium.png|0| Removed|
|Switch Converged.With label wrapping.hover.chromium.png|0| Removed|
|Switch Converged.With label wrapping.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Default.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Default.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Large size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Large size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Small size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Small size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Subtle appearance.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default).hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected (default).pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Tab Selected.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and large size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - Dark Mode.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - High Contrast.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size - High Contrast.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical and small size.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.Vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical - RTL.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical - RTL.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon and vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only and vertical.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only and vertical.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon only.pressed.chromium.png|0| Removed|
|TabList and Tab Converged.With icon.hover.chromium.png|0| Removed|
|TabList and Tab Converged.With icon.pressed.chromium.png|0| Removed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.default.hover row.chromium.png|0| Removed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|0| Removed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|0| Removed|
|Table layout flex - headers.sortable.hover header.chromium.png|0| Removed|
|Table layout flex - headers.sortable.press header.chromium.png|0| Removed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.always visible.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.default.hover row.chromium.png|0| Removed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|0| Removed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|0| Removed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|0| Removed|
|Table layout table - headers.sortable.hover header.chromium.png|0| Removed|
|Table layout table - headers.sortable.press header.chromium.png|0| Removed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-darker.focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-darker.hover.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-lighter.focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- filled-lighter.hover.chromium.png|0| Removed|
|Textarea Converged.Appearance- outline (default).focused.chromium.png|0| Removed|
|Textarea Converged.Appearance- outline (default).hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-darker.hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- filled-lighter.hover.chromium.png|0| Removed|
|Textarea Converged.Invalid- outline.focused.chromium.png|0| Removed|
|Textarea Converged.Invalid- outline.hover.chromium.png|0| Removed|
|Textarea Converged.With value.focused.chromium.png|0| Removed|
|Textarea Converged.With value.hover.chromium.png|0| Removed|
|Toast.Full Toast.Toast visible.chromium.png|0| Removed|
|Toast.Title Only.Toast visible.chromium.png|0| Removed|
|Toast.Without Subtitle.Toast visible.chromium.png|0| Removed|
|ToggleButton Converged.Checked.hover.chromium.png|0| Removed|
|ToggleButton Converged.Checked.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|0| Removed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Circular.hover.chromium.png|0| Removed|
|ToggleButton Converged.Circular.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - RTL.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Disabled.hover.chromium.png|0| Removed|
|ToggleButton Converged.Disabled.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Icon only.hover.chromium.png|0| Removed|
|ToggleButton Converged.Icon only.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Outline.hover.chromium.png|0| Removed|
|ToggleButton Converged.Outline.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary - Dark Mode.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary - Dark Mode.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary - High Contrast.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary - High Contrast.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary Checked.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary Checked.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary Disabled.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary Disabled.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Size large.hover.chromium.png|0| Removed|
|ToggleButton Converged.Size large.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Size small.hover.chromium.png|0| Removed|
|ToggleButton Converged.Size small.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Subtle Checked.hover.chromium.png|0| Removed|
|ToggleButton Converged.Subtle Checked.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Subtle Disabled.hover.chromium.png|0| Removed|
|ToggleButton Converged.Subtle Disabled.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Subtle.hover.chromium.png|0| Removed|
|ToggleButton Converged.Subtle.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Transparent Checked.hover.chromium.png|0| Removed|
|ToggleButton Converged.Transparent Checked.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Transparent Disabled.hover.chromium.png|0| Removed|
|ToggleButton Converged.Transparent Disabled.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Transparent.hover.chromium.png|0| Removed|
|ToggleButton Converged.Transparent.pressed.chromium.png|0| Removed|
|ToggleButton Converged.With icon after content.hover.chromium.png|0| Removed|
|ToggleButton Converged.With icon after content.pressed.chromium.png|0| Removed|
|ToggleButton Converged.With icon before content.hover.chromium.png|0| Removed|
|ToggleButton Converged.With icon before content.pressed.chromium.png|0| Removed|
|Toolbar Converged.Default.Button Pressed.chromium.png|0| Removed|
|Toolbar Converged.Default.Toggle On.chromium.png|0| Removed|
|Toolbar Converged.Large.Button Pressed.chromium.png|0| Removed|
|Toolbar Converged.Large.Toggle On.chromium.png|0| Removed|
|Toolbar Converged.Small.Button Pressed.chromium.png|0| Removed|
|Toolbar Converged.Small.Toggle On.chromium.png|0| Removed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|0| Removed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|0| Removed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|0| Removed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|0| Removed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|0| Removed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|0| Removed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>